src/main/webapp/projectDocument/ProjectDocument.js

```diff
@@ -0,0 +1,127 @@
+import { Component, Fragment } from 'react';
+import { Documents } from '../components/Documents'
+import { DocumentHandler } from "../util/ajax";
+import { ProjectKeyDocuments } from '../util/KeyDocuments';
+import { ConfirmationDialog } from "../components/ConfirmationDialog";
+import { h } from 'react-hyperscript-helpers';
+
+ class ProjectDocument extends Component {
+
+   constructor(props) {
+    super(props);
+    this.state = {
+      documentsCollection: [],
+      keyDocuments: [],
+      additional: [],
+      showDialog: false,
+      action: '',
+      uuid: ''
+    };
+  }
+
+   componentDidMount() {
+    this.getAttachedDocuments();
+  }
+
+   getAttachedDocuments = () => {
+    DocumentHandler.attachedDocuments(this.props.attachedDocumentsUrl, this.props.projectKey).then(resp => {
+      this.setKeyDocuments(JSON.parse(resp.data.documents));
+    }).catch(error => {
+      console.error(error);
```

Comment:
Note: We really need a way to handle JS errors for the user more gracefully than just console-ing them.